modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -66,6 +66,18 @@ TEST(Test_Darknet, read_yolo_voc)
     ASSERT_FALSE(net.empty());
 }
 
+TEST(Test_Darknet, read_filestorage_yolo_voc)
+{
+    std::ifstream ifile(_tf("yolo-voc.cfg").c_str());
+    std::stringstream buffer;
+    buffer << " " << ifile.rdbuf(); // FIXME: FileStorage drops first character.
+    FileStorage ofs(".xml", FileStorage::WRITE | FileStorage::MEMORY);
+    ofs.write("cfgFile", buffer.str());
+    FileStorage ifs(ofs.releaseAndGetString(), FileStorage::READ | FileStorage::MEMORY | FileStorage::FORMAT_XML);
+    Net net = readNetFromDarknet(ifs["cfgFile"]);
```

Comment:
@asciian, please remind me why you decided to replace `std::istream` to `cv::FileNode`? The last one seems to be more complicated here.

Reply:
@dkurt Because `std::istream` couldn't become the arguments of exporting functions and knew no way to implements the topic of this issue without `cv::FileNode`. If you know that, Please tell me.

Reply:
@dkurt Ah, I know another way. With raw string, we can implements that. But the way requires adding the function two arguments which are used for telling the length of content.